fri/src/folder.rs

```diff
@@ -0,0 +1,26 @@
+use ark_ff::FftField;
+use std::sync::Arc;
+
+pub struct MultiplicativeFriFolder;
+
+#[allow(dead_code)]
+impl MultiplicativeFriFolder {
+    pub fn next_layer_element_from_two_previous_layer_elements<F: FftField>(
+        f_x: &F,
+        f_minus_x: &F,
+        eval_point: &F,
+        x: &F,
+    ) -> F {
+        let x_inv = x.inverse().unwrap();
```

Comment:
Is this applied to many elements? You might want to use this:

https://docs.rs/ark-ff/latest/ark_ff/fields/fn.batch_inversion.html

Reply:
will fix this on next PR